run-tests.js

```diff
@@ -31,19 +31,35 @@ const keys = {
     '--php-async': false,    // run php async tests only
 }
 
+const exchangeSpecificFlags = {
+    '--sandbox': false,
+    '--verbose': false,
+    '--private': false,
+    '--privateOnly': false,
+}
+
```

Comment:
I might suggest them to be `undefined` ```suggestion
const exchangeSpecificFlags = {
    '--sandbox': undefined,
    '--verbose': undefined,
    '--private': undefined,
    '--privateOnly': undefined,
}
```
to avoid any confusion in thoughts.

Reply:
additionally, why not just using one of them instead of two (private vs privateOnly). I definitely think that we should support only one.

Reply:
@ttodua I think false represents it better because it means they are not selected and undefined might be a bit ambiguous

Reply:
> additionally, we might use one of them instead of two (private vs privateOnly). I think that we shouldnt have multiple aliases here, one is enough (maybe `--private`, adding that to arguments is already meaningful without need of additional `Only` suffix).

@ttodua they're not alias, they are different options,
if you provide --private -> means you want to run the public tests as usual plus the private ones
if you provide --privaOnly -> only runs private tests
if you don't provide any -> runs public only

Reply:
ah oh, my bad. I havent checked them in previous PR and thought they were same.